api/v1/apiErrors.js

```diff
@@ -13,126 +13,139 @@
  */
 'use strict';
 
-const apiErrors = require('errors');
+const errors = require('errors');
 
-apiErrors.create({
+errors.create({
+  scope: exports,
   code: 11010,
   name: 'RefocusApiError',
 });
 
 // ----------------------------------------------------------------------------
 // Validation Errors
 // ----------------------------------------------------------------------------
-apiErrors.create({
+errors.create({
+  scope: exports,
   code: 11100,
   status: 400,
   name: 'ValidationError',
-  parent: apiErrors.RefocusApiError,
```

Comment:
Previously, errors could be overwritten if another error was defined with the same name in a different file, because it defined the error on the global errors object, which is the same whether you require apiErrors, dbErrors, or redisErrors. This was causing discrepancies with error handling with the cache enabled vs disabled. This fixes it by restricting the scope to this module.

Reply:
Nice.